nativescript-angular/router/ns-location-strategy.ts

```diff
@@ -52,19 +79,21 @@ export class NSLocationStrategy extends LocationStrategy {
             return "/";
         }
 
-        const state = this.peekState(this.currentOutlet);
+        const state = this.currentOutlet && this.currentOutlet.peekState();
```

Comment:
Is it possible for this.currentOutlet to be null when this.currentUrlTree is not null?

Reply:
We set `currentUrlTree` inside `pushStateInternal` where all incoming urls are transformed into outlets . There will be always at least 1 Outlet set as `currentOutlet` . The otherwise should be possible only if `pushState()` is called from Angular with empty url param.